pkg/api/testapi/testapi.go

```diff
@@ -278,7 +280,15 @@ func init() {
 			externalTypes:        api.Scheme.KnownTypes(externalGroupVersion),
 		}
 	}
-
+	if _, ok := Groups[admissionregistration.GroupName]; !ok {
```

Comment:
The purpose is adding the admissionregistration group to testapi even if it's not specified in "KUBE_TEST_API". Otherwise when running test locally, pkg/api/serialization_test.go doesn't test the admissionregistration types.